applications/launchpad/gui-react/src/components/Tabs/styles.ts

```diff
@@ -35,18 +39,20 @@ export const Tab = styled.button<{ selected?: string; tab?: TabProp }>`
   align-items: center;
   transition: ease-in-out 300ms;
   &:hover {
-    background-color: ${({ theme }) => theme.backgroundSecondary};
+    background-color: ${({ theme, inverted }) =>
+      inverted ? '#141414' : theme.backgroundSecondary};
```

Comment:
I think that `#141414` can be replaced with `theme.inverted.backgroundSecondary`.

Reply:
yes!

Reply:
changed